From: Ben Hutchings <ben@decadent.org.uk>
Subject: x86/efi: Autoload efi-pstore
Date: Wed, 24 Dec 2014 17:44:06 +0100
Bug-Debian: https://bugs.debian.org/703363
Bug-Debian: https://bugs.debian.org/773309
Forwarded: no

efi-pstore should be auto-loaded on EFI systems, same as efivars.

--- a/drivers/firmware/efi/efi-pstore.c
+++ b/drivers/firmware/efi/efi-pstore.c
@@ -400,3 +400,4 @@ module_exit(efivars_pstore_exit);
 
 MODULE_DESCRIPTION("EFI variable backend for pstore");
 MODULE_LICENSE("GPL");
+MODULE_ALIAS("platform:efivars");
